### DISCUSSION TITLE
[Question] where to install a ".theia" plugin

### DISCUSSION THREAD

**Discussion by fiurthorn** [eclipse-theia/theia#9171] [Q&A]
I've created a the hello-world-backend-plugin und got a theia file.

I try to install the plugin in via the home folder. But i doesn't loaded.

VS Code have the option "install from VSIX" to install manual downloaded Extentions.

**Comment by akosyakov** [comment]
> I try to install the plugin in via the home folder. But i doesn't loaded.

it is not supported yet, certainly it would be interesting to support for `Electron` with custom product name, i.e. `~/.mytheiaproduct/plugins`. 

You can use `THEIA_DEFAULT_PLUGINS` to specify a directory with plugins, for instance `THEIA_DEFAULT_PLUGINS=local-dir:~/.mytheiaproduct/plugins`
